library/widget/tactic_component.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) E.W.Ayers. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+
+Author: E.W.Ayers
+-/
+import .basic
+
+meta def tactic.run_simple {α} : tactic_state → tactic α → option α
```

Comment:
Please move to `init.meta.tactic`.  I have the feeling we already have a handful of these functions.